packages/suite-desktop/src/support/DesktopUpdater/Downloading.tsx

```diff
@@ -65,13 +71,24 @@ const Downloading = ({ hideWindow, progress }: Props) => {
             onCancel={hideWindow}
         >
             <DownloadWrapper>
-                <Text>
-                    <Translation id="TR_DOWNLOADING" />
-                </Text>
-                <DownloadProgress>
-                    <ReceivedData>{toHumanReadable(progress?.transferred || 0)}</ReceivedData>/
-                    <TotalData>{toHumanReadable(progress?.total || 0)}</TotalData>
-                </DownloadProgress>
+                {progress?.verifying ? (
+                    <Text>
+                        <Translation id="TR_VERIFYING_SIGNATURE" />
+                        {ellipsisArray.filter((_, k) => k < step)}
+                    </Text>
+                ) : (
+                    <>
+                        <Text>
+                            <Translation id="TR_DOWNLOADING" />
+                        </Text>
+                        <DownloadProgress>
+                            <ReceivedData>
```

Comment:
lil' nitpick - there might be some sort of loading state if `transferred` and `total` are not available instead of showing `0 / 0`

Reply:
Good spot but out of scope for this PR. I will do some more testing and address it in a different PR if needed.